packages/webawesome/src/components/dropdown/dropdown.ts

```diff
@@ -109,6 +109,18 @@ export default class WaDropdown extends WebAwesomeElement {
 
   async updated(changedProperties: PropertyValues) {
     if (changedProperties.has('open')) {
+      const previousOpen = changedProperties.get('open');
+      // check if the previous value is the same
+      // (if they are, do not trigger menu showing / hiding)
+      if (previousOpen === this.open) {
+        return;
+      }
```

Comment:
this is the specific check that skips `hideMenu` or `showMenu` if we already are in the state we want to be in.

Reply:
Yeah! I'm happy to add a test here to verify that behavior 👍